commit 9834a31d037fc9df5043bd7b587aa7933313e917
Author: Phil Elwell <phil@raspberrypi.org>
Date:   Tue Jul 14 10:26:09 2015 +0100

    spi: spidev: Completely disable the spidev warning
    
    An alternative strategy would be to use "rpi,spidev" instead, but that
    would require many Raspberry Pi Device Tree changes.
    
    Signed-off-by: Phil Elwell <phil@raspberrypi.org>

diff --git a/drivers/spi/spidev.c b/drivers/spi/spidev.c
index b0c76e2626ce..b0657fdfe171 100644
--- a/drivers/spi/spidev.c
+++ b/drivers/spi/spidev.c
@@ -725,7 +725,7 @@ static int spidev_probe(struct spi_device *spi)
 	 * compatible string, it is a Linux implementation thing
 	 * rather than a description of the hardware.
 	 */
-	WARN(spi->dev.of_node &&
+	WARN(0 && spi->dev.of_node &&
 	     of_device_is_compatible(spi->dev.of_node, "spidev"),
 	     "%pOF: buggy DT: spidev listed directly in DT\n", spi->dev.of_node);
 
diff -uN a/1.txt b/1.txt
--- a/dummy/rpi_1010_9834a31d037fc9df5043bd7b587aa7933313e917.txt	1970-01-01 00:00:00.000000000 +0000
+++ b/dummy/rpi_1010_9834a31d037fc9df5043bd7b587aa7933313e917.txt	2013-12-23 04:07:40.000000000 +0000
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
